app/assets/stylesheets/book.css.scss

```diff
@@ -247,6 +247,10 @@
   text-align: center;
 }
 
+sub.beta-subscript{
```

Comment:
Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space